src/module_wrap.cc

```diff
@@ -279,8 +279,19 @@ void ModuleWrap::Link(const FunctionCallbackInfo<Value>& args) {
     Utf8Value specifier_utf8(env->isolate(), specifier);
     std::string specifier_std(*specifier_utf8, specifier_utf8.length());
 
+    Local<Object> assertions = Object::New(isolate);
+    Local<FixedArray> raw_assertions = module_request->GetImportAssertions();
+    for (int i = 0; i < raw_assertions->Length(); i += 3) {
+      assertions
+          ->Set(env->context(),
+                Local<String>::Cast(raw_assertions->Get(env->context(), i)),
+                Local<Value>::Cast(raw_assertions->Get(env->context(), i + 1)))
+          .ToChecked();
```

Comment:
```suggestion
      if (assertions
          ->Set(env->context(),
                Local<String>::Cast(raw_assertions->Get(env->context(), i)),
                Local<Value>::Cast(raw_assertions->Get(env->context(), i + 1)))
          .IsNothing()) { return; }
```

Reply:
maybe a null proto object would be more appropriate anyway?